frontend/packages/kubevirt-plugin/src/components/virtualization-overview/running-vms-per-template-card/RunningVMsPerTemplateCard.tsx

```diff
@@ -23,11 +23,12 @@ const getTemplateToVMCountMap = (resources) => {
   const templates = loaded && resources?.templates;
 
   const templateToVMCountMap = new Map();
-  const numVMs = vms.length;
+  const numVMs = vms?.length || 0;
 
   if (loaded) {
     vms.forEach((vm) => {
-      const template = vm?.metadata?.labels[LABEL_USED_TEMPLATE_NAME];
+      const labels = vm?.metadata?.labels;
+      const template = labels ? labels[LABEL_USED_TEMPLATE_NAME] : 'Other';
```

Comment:
This is where I thought the error may be yesterday when I looked at it. 

You should be able to just do `labels?.[LABEL_USED_TEMPLATE_NAME] || 'Other'`, which is a little cleaner in my opinion. At that point you can choose whether or not you want to keep the `labels`/`template` split.